pkg/apiclient/workflow/workflow.pb.go

```diff
@@ -806,6 +806,7 @@ var xxx_messageInfo_WorkflowDeleteResponse proto.InternalMessageInfo
 type WatchWorkflowsRequest struct {
 	Namespace            string          `protobuf:"bytes,1,opt,name=namespace,proto3" json:"namespace,omitempty"`
 	ListOptions          *v1.ListOptions `protobuf:"bytes,2,opt,name=listOptions,proto3" json:"listOptions,omitempty"`
+	Fields               string          `protobuf:"bytes,3,opt,name=fields,proto3" json:"fields,omitempty"`
```

Comment:
Still need to get rid of changes in this file. Doing a `git checkout master pkg/apiclient/workflow/workflow.pb.go` should suffice